src/3_PB/VerbaliInterni/VerbaleInterno_240223/content.typ

```diff
@@ -0,0 +1,41 @@
+#import "meta.typ": inizio_incontro, fine_incontro, luogo_incontro
+#import "functions.typ": glossary, team
+#let participants = csv("participants.csv")
+
+= Partecipanti
+/ Inizio incontro: #inizio_incontro
+/ Fine incontro: #fine_incontro
+/ Luogo incontro: #luogo_incontro
+
+#table(
+  columns: (3fr, 1fr),
+  [*Nome*], [*Durata presenza*],
+  ..participants.flatten()
+)
+= Sintesi Elaborazione Incontro
+Nell'incontro si è discusso sull'organizzazione del lavoro futuro stabilendo quanto riportato di seguito.
+
+== Nuovi ruoli
+Vengono stabiliti i seguenti i nuovi incarichi relativi allo sprint corrente:
+- Caregnato Simone: Responsabile;
+- Costantin Riccardo Alberto: Progettista;
+- D'Ovidio Giacomo: Progettista e Verificatore;
+- Kalaj Nancy: Verificatore e Amministratore;
+- Rango Matteo: Progettista;
+- Toniolo Riccardo: Progettista.
+Si decide di mantenere molti Progettisti in quanto svolgono il fulcro del lavoro di questo periodo, il ruolo va quindi utilizzato in queste fasi iniziali del PB e verrà gradualmente ridotto in seguito.
+
+== Stesura dei verbali odierni
+La stesura dei due verbali esterno ed interno sarà a carico di Simone Caregnato in qualità di responsabile.
```

Comment:
```suggestion
La stesura del verbale esterno ed del verbale interno saranno a carico di Simone Caregnato in qualità di responsabile.
```
Serve ribadirlo? da norme di progetto è la procedura standard

Reply:
Stesso discorso per il punto successivo

Reply:
Ho seguito un po' quello che aveva fatto Matteo nel verbale precedente, ma in effetti non serve